Dockerfile

```diff
@@ -1,15 +1,11 @@
 # syntax=docker/dockerfile:1.4
-FROM node:14.21.1-alpine3.15 AS base
+FROM node:18.15.0-alpine3.17 AS base
+ENV HUSKY 0
 
 RUN apk update
 RUN apk upgrade
 
-# installing specific python version based on your previous configuration
-RUN apk add --no-cache tini python2
-# installing specific make version based on your previous configuration
-RUN apk add make=4.2.1-r2 --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/main
-# installing specific gcc version based on your previous configuration
-RUN apk add g++=9.3.0-r0 --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/main
+RUN apk add --no-cache tini make g++ python3
```

Comment:
Why are we not specifying make and g++ version like before

Reply:
We don't need to be specific with those package versions anymore.

Reply:
Ok majorly these were need as part of isolate vm I see we are updating python 3 (should be fine if we are testing these)